### PR TITLE
Updates p-link--inverted to have a parent with is-dark

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates the following to have a 'is-dark' parent, so these need to be checked that they are no longer broken. The other changed pages were just formatting as I check them.

https://ubuntu-com-13745.demos.haus/kubernetes
https://ubuntu-com-13745.demos.haus/dell
https://ubuntu-com-13745.demos.haus/blog/topics/snapcraft
https://ubuntu-com-13745.demos.haus/blog/topics/maas
https://ubuntu-com-13745.demos.haus/blog/topics/juju
https://ubuntu-com-13745.demos.haus/blog/topics/design
https://ubuntu-com-13745.demos.haus/blog/design
https://ubuntu-com-13745.demos.haus/azure/fips
https://ubuntu-com-13745.demos.haus/16-04
https://ubuntu-com-13745.demos.haus/ (specifically the fallback takeover)
https://ubuntu-com-13745.demos.haus/ (normal takeovers)
https://ubuntu-com-13745.demos.haus/kubernetes/resources
https://ubuntu-com-13745.demos.haus/telco
https://ubuntu-com-13745.demos.haus/wsl

## QA

- Go to the above pages and find a section with inverted link, they should now be white on a dark background. 
Example on /dell, before:
image
After:
image

### CODE REVIEW

File: templates/16-04/index.html [hunk 1]
```diff
@@ -1,324 +1,383 @@
...
+        </p>
+        <p>
+          ESM is available for free on up to three machines and for broader enterprise use through an Ubuntu Advantage subscription. The subscription includes additional services such as FIPS-compliant modules and the Ubuntu Livepatch Service to apply critical kernel patches without unplanned downtime.
+        </p>
+        <p>
+          <a href="/security/livepatch">Learn more about Ubuntu Livepatch&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+      <div class="col-6 u-hide--medium u-hide--small u-align--center u-vertically-center">
+        {{ image (
```
[the remark this answers is not in the document]
Comment (akbarkz) [reply templates/16-04/index.html:126]: It's the opposite, we agreed on removing the space and thus letting djLint format it in a way that the `image` function name has a different indentation from its properties'.

**Comment by akbarkz** [comment]
Your lint-jinja CI action is failing, please fix formatting in one of the files

File: templates/16-04/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/16-04/index.html:126]: that is what I saying. But why add an extra step that has to be done manually like this? (removing the space)

[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/16-04/index.html:126]: sometimes it formats like this:
image
other times like this:
image

**Review by akbarkz — APPROVED** [review]

**Comment by anthonydillon** [comment]
@petesfrench this PR is tackling far more than it should for a single atomic change. I was alerted to it as it has reverted a change delivered in https://github.com/canonical/ubuntu.com/pull/13747/files which has introduced a broken link which is visible in our daily CI. 

Can we ensure that all PRs are rebased before merging and that we keep PRs atomic to catch these issues before they hit production please? 

CC: as reviewer @akbarkz

**Comment by petesfrench** [comment]
@anthonydillon Agreed, let's not let this happen again

**Pull request merged**